swupd/format_switch.go

```diff
@@ -61,6 +61,17 @@ includes:	{{.Name}}
 `,
 }
 
+const statusExperimental = "Experimental"
```

Comment:
Where does this string comes from? Swupd?

Reply:
It comes from bundle header.
"[Status] : Experimental"

Reply:
I mean, who sets that value?

Reply:
Whoever creates/edits the bundle.